connector/protobuf/src/test/scala/org/apache/spark/sql/protobuf/ProtobufCatalystDataConversionSuite.scala

```diff
@@ -24,6 +24,7 @@ import org.apache.spark.sql.{RandomDataGenerator, Row}
 import org.apache.spark.sql.catalyst.{CatalystTypeConverters, InternalRow, NoopFilters, OrderedFilters, StructFilters}
 import org.apache.spark.sql.catalyst.expressions.{ExpressionEvalHelper, GenericInternalRow, Literal}
 import org.apache.spark.sql.catalyst.util.{ArrayBasedMapData, GenericArrayData, MapData}
+import org.apache.spark.sql.protobuf.protos.CatalystTypes.BytesMsg
```

Comment:
I am struggling in generating this file in IDEA:
<img width="1027" alt="Screenshot 2022-10-28 at 17 02 04" src="https://user-images.githubusercontent.com/1580697/198630233-33341e69-276f-4505-9d52-874ddd1e6bc1.png">

Is any way to generate all required dependencies?

Reply:
@MaxGekk did you import it as maven or sbt project? Both should work though. 
With maven, 'generating sources' should fix that (in maven 'Tool Window'). I am also seeing similar issue with it, IntelliJ does not seem to run the `generate-test-sources` phase. 

I tried just now and running `../../build/mvn test` in protobuf directory fixed this issue. 

<img width="582" alt="image" src="https://user-images.githubusercontent.com/502522/198683965-67b5c95f-f1e6-4a9b-9f96-1607026946a0.png">